src/driver.ts

```diff
@@ -69,6 +69,24 @@ type DriverState = {
   descriptiveRunName: string;
 };
 
+/**
+ * A simple wrapper around `child_process.execFileSync` that adds additional details to the  exception message, if any.
+ */
+export function execFileSyncWithImprovedExceptionMessage(
+  bin: File,
+  args?: ReadonlyArray<string>,
+  options?: cp.ExecFileSyncOptions | cp.ExecFileSyncOptionsWithBufferEncoding
+) {
+  try {
+    return cp.execFileSync(bin, args, options);
+  } catch (e) {
+    let msg = `Failed to execute child-process in ${
+      options.cwd ? options.cwd : path.resolve(".")
```

Comment:
```suggestion
      options && options.cwd ? options.cwd : path.resolve(".")
```

`options` is optional, so this might cause a null-pointer.